parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestInputOutputFormatWithPadding.java

```diff
@@ -190,6 +189,23 @@ public void testBasicBehaviorWithPadding() throws Exception {
     ParquetFileWriter.BLOCK_FS_SCHEMES.remove("file");
   }
 
+  private List<String> readAllLines(File file) throws IOException {
```

Comment:
maybe we can put this in a shared file so it's not duplicated, maybe in org.apache.parquet.Files, next to org.apache.parquet.Strings?